From: Antoine Musso <hashar@free.fr>
Date: Fri, 29 Jul 2016 10:57:53 +0200
Subject: WMF: reduce Gerrit replication delay from 300 to 15 secs

Monkey patching to prevent Zuul from being deadlocked for 300 seconds
whenever a patch is merged.

Bug: T93812
Change-Id: I76943fff8f5d843dd9b13e8b92d71a002f99249e
---
 zuul/source/gerrit.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/zuul/source/gerrit.py b/zuul/source/gerrit.py
index 463f315..7cc0b3d 100644
--- a/zuul/source/gerrit.py
+++ b/zuul/source/gerrit.py
@@ -37,7 +37,7 @@ def detect_cycle(change, history=None):
 class GerritSource(BaseSource):
     name = 'gerrit'
     log = logging.getLogger("zuul.source.Gerrit")
-    replication_timeout = 300
+    replication_timeout = 15
     replication_retry_interval = 5
 
     depends_on_re = re.compile(r"^Depends-On: (I[0-9a-f]{40})\s*$",
